hack/deploy_vault_config.sh

```diff
@@ -200,6 +200,28 @@ getUserIDByLDAPName() {
 getUserIDByLDAPName brawilli
 vault write identity/group name="release-controller" policies="release-controller" member_entity_ids="$(getUserIDByLDAPName brawilli)"
 
+vault policy write vault-secret-collection-manager -<<EOH
```

Comment:
Something with these is still wrong, but its hard to test when the deployment continously overwrites :upside_down_face: